acl-lib/src/cmr/acl/acl_fetcher.clj

```diff
@@ -72,12 +98,19 @@
   caller is responsible for catching and logging the exception."
   [context]
   (let [cache (cache/context->cache context acl-cache-key)
-        updated-acls (echo-acls/get-acls-by-types
+        updated-acls (process-search-for-acls
                        ;; All of the object identity types needed by the application are fetched. We want
                        ;; the cache to contain all of the acls needed.
                        context (context->cached-object-identity-types context))]
     (cache/set-value cache acl-cache-key updated-acls)))
 
+(comment
```

Comment:
If you think this comment block is useful to keep around I'd move it to the bottom of the file.